Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,46 @@
+# Microcontroller Signoff
+## FUNCTION OF THE SUBSYSTEM
+The microcontroller and its code are for operating, managing, communicating with, maintaining, and collaborating with all of the other subsystems and acts as both the brain and body of the device.  All the data and instructions will through the microcontroller making it especially important to complete first and giving it a high level of importance.  The hardware will be responsible for the actual actions and completion of any given task.  The code will be responsible for controlling and giving the hardware tasks and manipulating data throughout the system and device.  
+
+Within this subsystem, the code will instruct the hardware in how to sample from the sensors, send data to the Communication subsystem, store data onto a USB Flash drive, and monitor battery voltage levels for the user.  These are the core functions that are desired from the microcontroller.  The microcontroller must be a reasonable portion of the end-user's budget of $300.
+
+## CONSTRAINTS ON THE SUBSYSTEM
+The constraints given below are generalized to allow for comparison of any potential contender for the microcontroller.  Constraints on the subsystem are as follows:
+1. The chosen microcontroller should cost no more than $60 as the end-user only has a budget of $300.  This board value cap is gathered from estimating the costs of other subsystems and viewing prices of available options such as Raspberry Pis, Arduinos, Intel, ESP, etc.  Up to 20% of the budget was then allocated due to the importance of this subsystem.
```

Comment:
average cost of market systems is not a constraint. Nothing and no-one requires this. So, it is not a constraint.

Reply:
This is a customer requirement and apart of our shall statements to make the device meet a cheap budget.  It is a constraint.